.changeset/fair-oranges-love.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fix `declaration-block-no-redundant-longhand-properties` false positives for `inherit` keyword
```

Comment:
[suggest] Thanks for adding the changelog. In addition, could you please use the standard prefix `FIXED:`? Please see also the [maintainer guide](https://github.com/stylelint/stylelint/blob/d171517376238ff2ba33f98e9e171ee13ca0cd81/docs/maintainer-guide/pull-requests.md#merging).

```suggestion
Fixed: `declaration-block-no-redundant-longhand-properties` false positives for `inherit` keyword
```

Reply:
Thanks. I fixed the prefix. https://github.com/stylelint/stylelint/pull/6419/commits/117e5f63aa66a80e58ef28223fff0bace1c5b34a